order/src/main.cpp

```diff
@@ -0,0 +1,41 @@
+/*Copyright 2019 Anders Madsen, Emil Jørgensen Njor, Emil Stenderup Bækdahl, Frederik Baymler
+ *Mathiesen, Nikolaj Jensen Ulrik, Simon Mejlby Virenfeldt
+ *
+ *Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ *associated documentation files (the "Software"), to deal in the Software without restriction,
+ *including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ *sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ *furnished to do so, subject to the following conditions:
+ *
+ *The above copyright notice and this permission notice shall be included in all copies or
+ *substantial portions of the Software.
+ *
+ *THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ *NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ *DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT
+ *OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "order/generation_service.hpp"
+#include "order/order.hpp"
+#include "order/random_generator.hpp"
+#include <iostream>
+#include <vector>
+
+using namespace order;
+
+int main(int argc, char *argv[])
+{
+    std::vector<int> stations;
+
+    for (int i = 1; i < argc; ++i) {
+        stations.push_back(atoi(argv[i]));
+    }
+
+    RandomGenerator generator = RandomGenerator{stations};
+    GenerationService service = GenerationService{5555, generator};
```

Comment:
Denne port bruger wall clock allerede.
```suggestion
    GenerationService service = GenerationService{7777, generator};
```

Reply:
Hvorfor ikke bare 0?

Reply:
Fordi så skal vi til at gætte hvilken port den er på. I stedet for at vi ved den er på en bestemt port.